requirements/constraints.txt

```diff
@@ -256,80 +256,6 @@ entrypoints==0.3 \
 websocket_client==0.57.0 \
     --hash=sha256:0fc45c961324d79c781bab301359d5a1b00b13ad1b10415a4780229ef71a5549 \
     --hash=sha256:d735b91d6d1692a6a181f2a8c9e0238e5f6373356f561bb9dc4c7af36f452010
-grpc-google-iam-v1==0.12.3 \
-    --hash=sha256:0bfb5b56f648f457021a91c0df0db4934b6e0c300bd0f2de2333383fe958aa72
```

Comment:
I need to check what other packages got added for pub/sub and remove those, too.